src/index.md

```diff
@@ -227,6 +241,20 @@ const tidy = clone(SparkRetrievalResultCodes).flatMap(({ day, rates }) => {
       ]
     })}
   </div>
+  <div class="card">
+      ${Plot.plot({
+      title: 'Time to First Byte (ms)',
+      x: { type: 'utc', ticks: 'day' }, // TODO: Change tick to month once we have more data
+      y: { grid: true },
```

Comment:
Can you please configure the chart so that the Y axis starts from `0` and the values are shown with zero decimal places?

Right now, it starts from `1,052.0`, which makes it look like there is a huge change in TTFB, while in reality, the change was only ~0.5%.

<img width="702" alt="Screenshot 2025-01-16 at 15 31 50" src="https://github.com/user-attachments/assets/0bce33ac-88da-4541-838e-52154e0854b3" />